### PR TITLE
Feat/era plugin

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Resource for provision of database

### CODE REVIEW

File: nutanix/common_era_schema.go [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	era "github.com/terraform-providers/terraform-provider-nutanix/client/era"
+)
+
+func timeMachineInfoSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeSet,
+		MaxItems:    1,
+		ForceNew:    true,
```
Comment (siddharth-kulshrestha) [inline nutanix/common_era_schema.go:14]: This schema is not getting used anywhere. 
Also, we have marked it as ForceNew. Make sure that the caller funcs calling this func don't provision the parent resources after changing the machine details.
Please check the behavior of Force new or make it as a parameter to this function.

File: nutanix/common_era_schema.go [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+		Optional:    true,
+		Description: "sample description for time machine info",
```
Comment (siddharth-kulshrestha) [inline nutanix/common_era_schema.go:16]: change this

Comment (siddharth-kulshrestha) [reply nutanix/common_era_schema.go:16]: Please make the changes everywhere.

File: nutanix/common_era_schema.go [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+					Type:        schema.TypeString,
+					Description: "",
+					Required:    true,
+				},
+			},
+		},
+	}
+}
+
+func tryToConvertBool(v interface{}) (bool, bool) {
```
Comment (siddharth-kulshrestha) [inline nutanix/common_era_schema.go:394]: Can we use the same ConvToBool func ?

**Review by siddharth-kulshrestha — CHANGES_REQUESTED** [review]

File: nutanix/common_era_schema.go (hunk 3 above)
Comment (Abhishekism9450) [reply nutanix/common_era_schema.go:394]: This func is no longer used. Removed it

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: success
>Current line coverage (6.8) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (7.8) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: success
>Current line coverage (8.3) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test pc
>Acceptance test run status: failure
>Current line coverage (40.1) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: failure
>Current line coverage (8.0) is below threshold (50). Kindly add more acceptance tests.

**Comment by abhimutant** [comment]
/ok-to-test era
>Acceptance test run status: success
>Current line coverage (8.3) is below threshold (50). Kindly add more acceptance tests.

**Review by siddharth-kulshrestha — APPROVED** [review]

**Pull request merged**